### ISSUE TITLE
Feature Request: Simple map projections

### ISSUE THREAD

**Issue by guidov** [mpld3/mpld3#89]
I'm curious if it would be possible in the future to implement mpld3 plotting onto a cartographic map projections such as is done with cartopy or basemap but using d3 (e.g. https://github.com/d3/d3-geo-projection ). 
For example, may people in the Earth Sciences do a lot of contouring of data over map projections. It would be amazing to be able to have interactive visualization of these type of matplotlib plots. 
Would this be possible in the future with mpld3?

**Comment by isokrates** [comment]
I would also love this! I use Basemap with Matplotlib to create maps with lines (great circles) between various nodes, and in iPython you can zoom right in on areas of interest, which is awesome. However, I want this to be a web app, so currently my only option is to output static figures and serve them. I would love to be able to show a dynamic map interface that users could zoom in to. Is this currently possible, or something that might be implemented at some point?

Thanks!

**Comment by jakevdp** [comment]
Certainly possible, and could be implemented if someone puts their mind to it! It would involve extending mpld3's JS backend to allow for different coordinate transforms, which would be rather difficult I think.

**Comment by aflaxman** [comment]
I think the way to do this is to push the projection work into `dj3s`, which has some really nice map projections implemented. @isokrates can you put together a minimal example of the kind of thing you do in python, and we can start from there to consider how to make interactive maps in `mpld3`?

**Comment by guidov** [comment]
Hi, here is an example hack of what I was originally interested with mpld3
and maps projections.  I think this kind of data inspection on the map
projection is quite useful.

http://nbviewer.ipython.org/github/guidov/mpld3-contour-basemap/blob/master/mpld3-contour-basemap.ipynb

But to have this on the d3 interactive ( http://bl.ocks.org/mbostock/3795040
) would be amazing.

**Comment by MaxPowerWasTaken** [comment]
any update on this? Is it possible/supported to use mpld3 to create map-based visualizations? I don't see any in the [gallery](https://mpld3.github.io/examples/index.html) or from a quick google (which led me here) but wanted to ask before writing it off in case I missed something. 

Thanks!

**Comment by Raz0rGithub** [comment]
Yes, this would be a wonderful feature to have. Thank you to the devs of mpld3.
